### PR TITLE
Added a failing test for #423

### PR DESCRIPTION
**Pull request by koenweyn**

### CODE REVIEW

File: src/directives/formly-field.test.js [hunk 1]
```diff
@@ -1014,6 +1014,32 @@ describe('formly-field', function() {
...
+          formatters: ['"!" + $viewValue + "!"']
+        })];
+        compileAndDigest();
+
+        const ctrl = getNgModelCtrl();
+
+        //workaround: this formats the model value right off the bat,
+        //without the workaround, the initial model value stays unformatted
+
+/*
```
Comment (kentcdodds) [inline src/directives/formly-field.test.js]: What's this commented block for? If it's supposed to be part of the test, then just leave it in. The contributor shouldn't have to alter anything except the `.skip` in this file :-) We want it to be as easy and straightforward as possible: "Just make the tests pass"

**Comment by kentcdodds** [comment]
Thanks a ton for doing this @koenweyn! This is super cool! If you could just address the one thing that I mentioned then I think we're good.

File: src/directives/formly-field.test.js (hunk 1 above)
Comment (koenweyn) [inline src/directives/formly-field.test.js]: The comment block is the workaround, I'll remove it and add it to the issue comments in stead.

**Comment by kentcdodds** [comment]
Thanks a ton!

**Pull request merged**